crates/vault-registry/src/lib.rs

```diff
@@ -323,7 +334,7 @@ decl_module! {
 #[cfg_attr(test, mockable)]
 impl<T: Config> Module<T> {
     fn begin_block(_height: T::BlockNumber) -> DispatchResult {
-        Self::liquidate_undercollateralized_vaults();
+        Self::liquidate_undercollateralized_vaults(LiquidationTarget::NonOperatorsOnly)?;
```

Comment:
Because Operator liquidation logic involves liquidating Nominators too, and since the pallet dependency is `nomination -> vault_registry`, it needs to be done in the `nomination` pallet